src/domUpdates.js

```diff
@@ -37,19 +38,39 @@ const domUpdates = {
     });
   },
 
-  populateTotalBill(num) {
+  populateAvailableRooms(hotel) {
+    availableRoomsSection.innerHTML = '';
+    const rooms = hotel.getAvailableRooms(dateInput.value);
+    rooms.forEach(room => {
+      const bidet = room.bidet ? 'yes' : 'no';
```

Comment:
This is a very neat trick for producing a more semantic string instead of "true" or "false"